source/WebConfig/cosa_webconfig_apis.h

```diff
@@ -139,3 +139,8 @@ CosaDmlRemoveConfigFileEntry
 
 
 int initConfigFileWithURL(char *Url, ULONG InstanceNumber);
+
+void CosaDmlRemoveValueFromDb(char *ParamName);
+void CosaDmlStoreValueIntoDb(char *ParamName, char *pString);
+void CosaDmlGetValueFromDb( char* ParamName, char* pString );
+void FillEntryInList(PCOSA_DATAMODEL_WEBCONFIG pWebConfig, PCOSA_DML_WEBCONFIG_CONFIGFILE_ENTRY configFileEntry);
```

Comment:
Check all above functions and only if needed expose in header

Reply:
Incorporated.